game.py

```diff
@@ -170,8 +196,12 @@ def new_game(self, game=None, restart=True):
             self._game = game
             self._level = 3
             self._correct = 0
+            self._correct_for_level = 0
         if restart:
-            self._all_clear()
+            if ( self._correct < 10): 
```

Comment:
Redundant parenthesis can be removed :smile: 

```suggestion
            if self._correct < 10: 
```